pkg/util/trivy/db.go

```diff
@@ -25,24 +25,6 @@ const (
 // onDeleteCallback describes a callback function that is called before deleting an entry from a PersistentDB.
 type onDeleteCallback = func(key string, value []byte) error
 
-// PersistentDB describes an interface for a persistent key-value store.
-type PersistentDB interface {
```

Comment:
Only BoltDB implements this interface. Thus I see no value in keeping it except for risks of `nil` pointers deref.